src/chunker/chunker.js

```diff
@@ -104,8 +103,10 @@ class Chunker {
 		this.hooks.layout = new Hook(this);
 		this.hooks.renderNode = new Hook(this);
 		this.hooks.layoutNode = new Hook(this);
+		this.hooks.getOverflow = new Hook(this);
```

Comment:
Could you add these new hooks to the list in the Readme?

Reply:
Sure!

Reply:
Removed the hook now - it ended up being an unneeded addition.